tests/Test1.robot

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Library           SeleniumLibrary
+Library           Process
+Library    XML
+Library    OperatingSystem
+
+*** Variables ***
+${URL}            http://decenterai.com
+${BROWSER}        chrome
+
+*** Test Cases ***
+
+Test Head
+    Log To Console    test_head
+    Open Browser    ${URL}    ${BROWSER}
+    Sleep    5
+    Page Should Contain  DecentralIzed and DemocratIzed AI 
+    Page Should Contain  Model TraInIng Infrastructure
+    Sleep    2
+
+
+    Page Should Contain Button  Try Demo
```

Comment:
can u raise a pr for the same change over DecenterAI/decenterai.com. 

This is the repo for app.decenterai.com. And pleace replace the url with localhost:3000

Reply:
alr, got it

Reply:
then do it